spec/factories/tasks.rb

```diff
@@ -0,0 +1,8 @@
+FactoryGirl.define do
+  factory :task do
+    project nil
```

Comment:
Since a task needs to have a project I would not override with `nil`. So a simple `project` would be good a get a default project associated.